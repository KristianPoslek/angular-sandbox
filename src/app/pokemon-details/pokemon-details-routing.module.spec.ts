import { routes } from './pokemon-details-routing.module';

describe('pokemonDetails.PokemonDetailsRoutingModule', () => {
    it('should define the routes', () => {
        expect(routes).toMatchSnapshot();
    });
});
