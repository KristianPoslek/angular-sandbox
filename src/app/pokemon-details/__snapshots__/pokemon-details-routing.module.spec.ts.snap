// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pokemonDetails.PokemonDetailsRoutingModule should define the routes 1`] = `
Array [
  Object {
    "component": [Function],
    "path": ":name",
  },
]
`;
